### ISSUE TITLE
/bat lookup [Name] Formatting

### ISSUE THREAD

**Issue by Blacksheep92** [alphartdev/BungeeAdminTools#34]
Since Minecraft 1.8 there is the possibility to use [Hover-/Click-Events.](www.spigotmc.org/wiki/the-chat-component-api/)

Lookup profile:

```
 ---- Statisiken Picard68 ----
Offline
Aktueller Status : 
  Gebannt auf : (global)
  IP-Gebannt auf : keine
  SGemutet auf : keine
  IP-Gemutet auf : keine
Erster Login : 09-12-2015 um 18:09
Zuletzt online : 29-12-2015 um 12:47
Letzte IP : 00.000.00.000
Vergangenheit : 1 bans
            1 mutes
            0 kicks
Letzten drei Bemerkungen: 
  #120 Warning Inhalt: nicht nachgepflanzt in einer stadt 2 Datum: 22-12-2015 um 18:57 Erstellt von: xhera#112 Warning Inhalt: Nicht nachgepflanzt 1 Datum: 19-12-2015 um 18:26 Erstellt von: Blacksheep92#111 Warning Inhalt: Griefing in Minera 4 Datum: 19-12-2015 um 17:51 Erstellt von: Blacksheep92
Vergangene Namen : Picard68
UUID : d5980c48624c45628b26b2ca9d4fd166
---- Statistiken Picard68 ----
```

My suggestions:
- Add a ClickEvent for "---- Statisiken Picard68 ----" to visit the website
- Add a HoverEvent for "---- Statisiken Picard68 ----" with UUID, LastNames, IP
- Add a HoverEvent for "Offline" with firstlogin & lastlogin
- Add the reason, time & "creator" of an ban/mute as a HoverEvent  
- Warning Example
  - Text:
  - #120 Reason: nicht nachgepflanzt in einer stadt 2
  - As HoverEvent: 
  - Datum: 22-12-2015 um 18:57 
  - Erstellt von: Hera
  - And add line breaks ;)

**Comment by alphartdev** [comment]
That's a good idea!

**Comment by cindyker** [comment]
Add the reason, time & "creator" of an ban/mute as a HoverEvent
This would work for my Feature request. 👍

**Comment by alphartdev** [comment]
Alright, so I'm working on that. I've currently implemented the HoverEvent.

The difficulty is to let the lookup customizable (so less hardcoded formatting) and at the same time provide enough information so it's useful. I'll give you a screenshot of what it looks like in the following days

**Comment by alphartdev** [comment]
So the hover has been implemented and uploaded in 1.5.2-premium version (https://www.spigotmc.org/resources/bungee-admin-tools-premium-edition.37880/update?update=174832)

By default, the current punishment are displayed with the hover.

For that part: 

> Add a HoverEvent for "---- Statisiken Picard68 ----" with UUID, LastNames, IP
> Add a HoverEvent for "Offline" with firstlogin & lastlogin

You can entirely customize it yourself, just look at the emsages_en.language file to see how to use the {effect="hover" TAG.

Finally, I'm planning to do some modifications for the webinterface (especially for the design)
> Add a ClickEvent for "---- Statisiken Picard68 ----" to visit the website
so that will be done at the same time

For the warnings, I'm waiting to get some feedback on what I've done for bans, mutes hover.

If you have any other suggestions or bug report, I'll be glad to hear them.
Again thanks for the idea, it really improves the lookup :)
